src/FieldTrait.php

```
@@ -20,6 +20,44 @@ trait FieldTrait {
+
+  /**
+   * Assert that a field is not empty.
+   *
+   * @code
+   * Then the field "Name" is not empty
+   * @endcode
+   *
+   * @Then the field :field should not be empty
+   */
+  public function fieldAssertNotEmpty(string $field): void {
+    $field_element = $this->fieldAssertExists($field);
```

PHPMD (2.15.0)

The variable $field_element is not named in camelCase. (undefined)

(CamelCaseVariableName)